Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1481,3 +1489,26 @@ extension PurchasesOrchestrator {
     }
 
 }
+
+// MARK: - Application Lifecycle
+extension PurchasesOrchestrator {
+    func handleApplicationDidBecomeActive() {
```

Comment:
total nitpick, but to match [our Style Guide](https://github.com/RevenueCat/purchases-ios/blob/main/Contributing/SwiftStyleGuide.swift) we should have line breaks right after the start and before the end of these definitions